node/index.ts

```diff
@@ -7,7 +7,8 @@ import * as os from 'os';
 import bindings = require('bindings'); // eslint-disable-line @typescript-eslint/no-require-imports
 import * as SignalClient from './libsignal_client';
 
-const SC = bindings('libsignal_client_' + os.platform()) as typeof SignalClient;
+const arch = process.env.npm_config_arch || process.arch
```

Comment:
Under what conditions will these be different?

Reply:
Good question. For the build step `npm_config_arch` is definitely needed to support cross-compilation from a x64 host. But since the code here will only run on the target platform (I assume), `process.env.npm_config_arch` should be superfluous here. I'll remove it and only use `process.arch` instead. Thanks!